### ISSUE TITLE
Modpack Initializing Crash - "java.lang.IllegalAccessError"

### ISSUE THREAD

**Issue by pyroAvatar91** [DimensionalDevelopment/JustEnoughIDs#186]
I'm not sure if this is actually related to JEIDs, but a google search led me to here with similar looking crash reports. Please let me know if I should be directing this issue elsewhere. I've attached my crash report below. I have the latest version of JEID for 1.12.2 installed.

I have no experience coding, but it looks like it might be a mixin problem. I attempted to use the "Mixin 0.7-0.8 Compatibility" mod by NotStirred but that didn't seem to solve it. I know there's a mod called "MixinBootstrap" by LX_Gaming but haven't tried that yet as I'm not sure that is the right thing to try.

I've tried disabling FoamFix as I understand that can often be a problem, but that didn't fix anything. Please let me know if you need the Debug log or any other information. This crash report seems to me it is a bit sparse on information. This may be due to the crash occurring about 30 seconds after initializing. 

I appreciate the help! Thanks!

---- Minecraft Crash Report ----
// Why is it breaking :(

Time: 2021-12-29 12:33:14 MST
Description: Initializing game

java.lang.IllegalAccessError: tried to access class org.spongepowered.asm.mixin.transformer.MixinProcessor from class net.minecraftforge.fml.common.Loader
    at net.minecraftforge.fml.common.Loader.handler$zzb001$beforeConstructingMods(Loader.java:1641)
    at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:594)
    at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:232)
    at net.minecraft.client.Minecraft.init(Minecraft.java:467)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3931)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Windows 10 (amd64) version 10.0
  Java Version: 1.8.0_51, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 234966480 bytes (224 MB) / 855638016 bytes (816 MB) up to 6375342080 bytes (6080 MB)
  JVM Flags: 21 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx6080m -Xms256m -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=200 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1
  IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
  FML: MCP 9.42 Powered by Forge 14.23.5.2860 Optifine OptiFine_1.12.2_HD_U_G5 142 mods loaded, 142 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State | ID                       | Version                 | Source                                              | Signature                                |
       |:----- |:------------------------ |:----------------------- |:--------------------------------------------------- |:---------------------------------------- |
       | L     | minecraft                | 1.12.2                  | minecraft.jar                                       | None                                     |
       | L     | mcp                      | 9.42                    | minecraft.jar                                       | None                                     |
       | L     | FML                      | 8.0.99.99               | forge-1.12.2-14.23.5.2860.jar                       | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | L     | forge                    | 14.23.5.2860            | forge-1.12.2-14.23.5.2860.jar                       | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | L     | creativecoredummy        | 1.0.0                   | minecraft.jar                                       | None                                     |
       | L     | entity_culling           | 4.2.5                   | minecraft.jar                                       | None                                     |
       | L     | littletilescore          | 1.0.0                   | minecraft.jar                                       | None                                     |
       | L     | foamfixcore              | 7.7.4                   | minecraft.jar                                       | None                                     |
       | L     | replaymod                | 1.12.2-2.6.1            | replaymod-1.12.2-2.6.1.jar                          | None                                     |
       | L     | main_mod                 | 1.0.0                   |  SoWatDiKhapVN's Modern Mod v1.5 (1.12.2) + Fix.jar | None                                     |
       | L     | securitycraft            | v1.9.0.2                | [1.12.2] SecurityCraft v1.9.0.2.jar                 | None                                     |
       | L     | creativecore             | 1.10.0                  | CreativeCore_v1.10.69_mc1.12.2.jar                  | None                                     |
       | L     | littletiles              | 1.5.0                   | LittleTiles_v1.5.57_mc1.12.2.jar                    | None                                     |
       | L     | alet                     | 1.0                     | A_Little_Extra_Tiles-1.0.23.jar                     | None                                     |
       | L     | aireducer                | 0.3.0                   | AIReducer-1.12.2-0.3.0.jar                          | None                                     |
       | L     | ambientsounds            | 3.0                     | AmbientSounds_v3.1.7_mc1.12.2.jar                   | None                                     |
       | L     | arcademod                | 2.1.5                   | arcademod-2.1.5-1.12.2.jar                          | None                                     |
       | L     | armourers_workshop       | 1.12.2-0.51.0           | Armourers-Workshop-1.12.2-0.51.0.jar                | None                                     |
       | L     | astikorcarts             | 1.12.2-0.1.2.8          | astikorcarts-1.12.2-0.1.2.8.jar                     | None                                     |
       | L     | badmobs                  | 1.1.40                  | BadMobs-1.12.2-1.1.40.jar                           | None                                     |
       | L     | baubles                  | 1.5.2                   | Baubles-1.12-1.5.2.jar                              | None                                     |
       | L     | betterbiomeblend         | 1.12.2-1.1.7-forge      | betterbiomeblend-1.12.2-1.1.7-forge.jar             | None                                     |
       | L     | better_diving            | 1.5.8                   | BetterDiving-1.12.2-1.5.8.jar                       | None                                     |
       | L     | betterthirdperson        | 1.5.4                   | BetterThirdPerson-Forge-1.12.2-1.5.4.jar            | None                                     |
       | L     | bookworm                 | 1.12.2-2.5.2.1          | bookworm-1.12.2-2.5.2.1.jar                         | None                                     |
       | L     | dynamictrees             | 1.12.2-0.9.26           | DynamicTrees-1.12.2-0.9.26.jar                      | None                                     |
       | L     | jeid                     | 1.0.4-SNAPSHOT          | JustEnoughIDs-1.0.4-SNAPSHOT-thin.jar               | None                                     |
       | L     | climaticbiomesjbg        | 2.16.4                  | ClimaticBiomes-2.16.4-MC1.12.2.jar                  | None                                     |
       | L     | climaticbiomesdt         | 1.4.0                   | CB-DTCompat-1.4.0-MC1.12.2.jar                      | None                                     |
       | L     | ctm                      | MC1.12.2-1.0.2.31       | CTM-MC1.12.2-1.0.2.31.jar                           | None                                     |
       | L     | jei                      | 4.16.1.301              | jei_1.12.2-4.16.1.301.jar                           | None                                     |
       | L     | chisel                   | MC1.12.2-1.0.2.45       | Chisel-MC1.12.2-1.0.2.45.jar                        | None                                     |
       | L     | chiselsandbits           | 14.33                   | chiselsandbits-14.33.jar                            | None                                     |
       | L     | codechickenlib           | 3.2.3.358               | CodeChickenLib-1.12.2-3.2.3.358-universal.jar       | None                                     |
       | L     | collective               | 3.0                     | collective-1.12.2-3.0.jar                           | None                                     |
       | L     | configuration            | 1.0.3.1                 | configuration-1.12.2-1.0.3.1.jar                    | None                                     |
       | L     | conquest                 | 3.0.2                   | ConquestReforged-3.0.2-mc1.12.2.jar                 | None                                     |
       | L     | blockpalette             | 1.5.0                   | ConquestReforged-3.0.2-mc1.12.2.jar                 | None                                     |
       | L     | connect                  | 1.0.3-mc1.12.2-SNAPSHOT | ConquestReforged-3.0.2-mc1.12.2.jar                 | None                                     |
       | L     | controlling              | 3.0.10                  | Controlling-3.0.10.jar                              | None                                     |
       | L     | cookingforblockheads     | 6.5.0                   | CookingForBlockheads_1.12.2-6.5.0.jar               | None                                     |
       | L     | corpse                   | 1.12.2-1.0.8            | corpse-1.12.2-1.0.8.jar                             | None                                     |
       | L     | modcurrency              | 1.12-1.3.11             | Currency-1.12.0-1.3.11.jar                          | None                                     |
       | L     | customnpcs               | 1.12                    | CustomNPCs_1.12.2-(05Jul20).jar                     | None                                     |
       | L     | customrecord             | 1.0                     | customrecord-1.12.2-1.1.jar                         | None                                     |
       | L     | cyclepaintings           | 2.0                     | cyclepaintings_1.12.2-2.0.jar                       | None                                     |
       | L     | fcl                      | 1.12.70                 | Fexcraft Common Library-1.12.70.jar                 | None                                     |
       | L     | trackapi                 | 1.2                     | TrackAPI-1.2.jar                                    | None                                     |
       | L     | fvtm                     | 3.7.58                  | FVTM-3.7.58-1.12.2.jar                              | None                                     |
       | L     | d33bmw                   | 0.1                     | D33BMW-FVTM-1.12.2-0.0.3.jar                        | None                                     |
       | L     | d33dodge                 | 0.1                     | D33DODGE-FVTM-1.12.2-0.0.1.jar                      | None                                     |
       | L     | d33lib                   | 0.1                     | D33LIB-FVTM-1.12.2-0.0.4.jar                        | None                                     |
       | L     | d33rolls                 | 0.1                     | D33Rolls-Royce-FVTM-1.12.2-0.0.1.1.jar              | None                                     |
       | L     | d33vaz                   | 0.1                     | D33VAZ-FVTM-1.12.2-0.0.2.jar                        | None                                     |
       | L     | doubledoors              | 3.0                     | doubledoors_1.12.2-3.0.jar                          | None                                     |
       | L     | doubleslabs              | 0.12.2                  | DoubleSlabs-1.12-0.12.2.jar                         | None                                     |
       | L     | harvestcraft             | 1.12.2zb                | Pam's HarvestCraft 1.12.2zg.jar                     | None                                     |
       | L     | dynamictreesphc          | 1.12.2-2.0.5            | DynamicTreesPHC-1.12.2-2.0.5.jar                    | None                                     |
       | L     | traverse                 | 1.6.0                   | Traverse-1.12.2-1.6.0-69.jar                        | None                                     |
       | L     | dttraverse               | 1.12.2-2.1              | DynamicTreesTraverse-1.12.2-2.1.jar                 | None                                     |
       | L     | effortlessbuilding       | 1.12.2-2.16             | effortlessbuilding-1.12.2-2.16.jar                  | None                                     |
       | L     | engineersdecor           | 1.1.3                   | engineersdecor-1.12.2-1.1.3.jar                     | None                                     |
       | L     | immersiveengineering     | 0.12-98                 | ImmersiveEngineering-0.12-98.jar                    | None                                     |
       | L     | engineersdoors           | 0.9.1                   | engineers_doors-1.12.2-0.9.1.jar                    | None                                     |
       | L     | exoticbirds              | 1.0                     | Exotic Birds 1.12.2-3.2.0.jar                       | None                                     |
       | L     | extrabitmanipulation     | 1.12.2-3.4.1            | ExtraBitManipulation-1.12.2-3.4.1.jar               | None                                     |
       | L     | farseek                  | 2.5.1                   | Farseek-1.12-2.5.1.jar                              | None                                     |
       | L     | fastleafdecay            | v14                     | FastLeafDecay-v14.jar                               | None                                     |
       | L     | tmt                      | 1.14                    | Fexcraft Common Library-1.12.70.jar                 | None                                     |
       | L     | fvp                      | 2.6.3                   | Fexcraft Vehicle Pack-2.6.3.jar                     | None                                     |
       | L     | foamfix                  | 0.10.14-1.12.2          | foamfix-0.10.14-1.12.2.jar                          | None                                     |
       | L     | forgelin                 | 1.8.4                   | Forgelin-1.8.4.jar                                  | None                                     |
       | L     | forgemultipartcbe        | 2.6.2.83                | ForgeMultipart-1.12.2-2.6.2.83-universal.jar        | None                                     |
       | L     | microblockcbe            | 2.6.2.83                | ForgeMultipart-1.12.2-2.6.2.83-universal.jar        | None                                     |
       | L     | minecraftmultipartcbe    | 2.6.2.83                | ForgeMultipart-1.12.2-2.6.2.83-universal.jar        | None                                     |
       | L     | fpsreducer               | mc1.12.2-1.20           | FpsReducer-mc1.12.2-1.20.jar                        | None                                     |
       | L     | ftbbackups               | 1.1.0.1                 | FTBBackups-1.1.0.1.jar                              | None                                     |
       | L     | futuremc                 | 0.2.6                   | future-mc-0.2.8.jar                                 | None                                     |
       | L     | fvtm_mts                 | 1.0                     | FVTM-MTS-1.0.jar                                    | None                                     |
       | L     | gep                      | 2.3.0                   | GEP-2.4.1c-1.12.2.jar                               | None                                     |
       | L     | harvest                  | 1.12-1.2.8-25           | Harvest-1.12-1.2.8-25.jar                           | None                                     |
       | L     | hcp                      | 2.1.3                   | HCP-2.1.3-1.12.2.jar                                | None                                     |
       | L     | udp                      | 1.0                     | HTP-1.0.jar                                         | None                                     |
       | L     | mts                      | 20.4.0                  | Immersive Vehicles-1.12.2-20.4.0.jar                | None                                     |
       | L     | immersivecables          | 1.3.2                   | ImmersiveCables-1.12.2-1.3.2.jar                    | None                                     |
       | L     | immersivepetroleum       | 1.1.10                  | immersivepetroleum-1.12.2-1.1.10.jar                | None                                     |
       | L     | immersiveposts           | 0.2.1                   | ImmersivePosts-0.2.1.jar                            | None                                     |
       | L     | universalmodcore         | 1.1.3                   | UniversalModCore-1.12.2-forge-1.1.3.jar             | None                                     |
       | L     | immersiverailroading     | 1.9.1                   | ImmersiveRailroading-1.12.2-forge-1.9.1.jar         | None                                     |
       | L     | immersiveruins           | 1.0.0                   | ImmersiveRuins-1.0.jar                              | None                                     |
       | L     | jeiintegration           | 1.6.0                   | jeiintegration_1.12.2-1.6.0.jar                     | None                                     |
       | L     | journeymap               | 1.12.2-5.7.1            | journeymap-1.12.2-5.7.1.jar                         | None                                     |
       | L     | jehc                     | 1.7.2                   | just-enough-harvestcraft-1.12.2-1.7.2.jar           | None                                     |
       | L     | justenoughpetroleum      | 0.1                     | JustEnoughPetroleum-0.1.jar                         | None                                     |
       | L     | kirosblocks              | 1.2.1                   | kirosblocks-1.2.1.jar                               | None                                     |
       | L     | landofsignals            | 0.0.4                   | LandOfSignals-1.12.2-forge-0.0.4-HOTFIX-1.jar       | None                                     |
       | L     | zawa                     | 1.12.2-2.1.3            | zawa-1.12.2-2.1.3.jar                               | None                                     |
       | L     | lilcritters              | 1.12.2-1.1.0.0          | lilcritters-1.12.2-1.1.0.0.jar                      | None                                     |
       | L     | limitlessstructureblocks | 1.1.0                   | LimitlessStructureBlocks-1.12.2-1.1.0.jar           | None                                     |
       | L     | littleframes             | 1.0.0                   | LittleFrames_v1.0.12_mc1.12.2.jar                   | None                                     |
       | L     | littleopener             | 1.0.1                   | littleopener-1.0.1.jar                              | None                                     |
       | L     | lt3dimporter             | 0.6.2                   | lt3dimporter-0.6.2.jar                              | None                                     |
       | L     | immersivetech            | 1.9.99                  | MCTImmersiveTechnology-1.12.2-1.9.99.jar            | None                                     |
       | L     | modnametooltip           | 1.10.1                  | modnametooltip_1.12.2-1.10.1.jar                    | None                                     |
       | L     | guilib                   | $version                | morepaintings-paintings-1.12.2-5.0.1.2.jar          | None                                     |
       | L     | paintingselgui           | $version                | morepaintings-paintings-1.12.2-5.0.1.2.jar          | None                                     |
       | L     | morepaintings            | $version                | morepaintings-paintings-1.12.2-5.0.1.2.jar          | None                                     |
       | L     | mousetweaks              | 2.10                    | MouseTweaks-2.10-mc1.12.2.jar                       | None                                     |
       | L     | movingelevators          | 1.2.34                  | movingelevators-1.2.34-mc1.12.jar                   | None                                     |
       | L     | more_police_items_mod    | 3.0                     | MPIM 1.12.2 BETA v.3.0.jar                          | None                                     |
       | L     | kadwinjpvehicles         | 2.0.0                   | MTS_Kadwin_JP_Vehicles_Pack_3.2.jar                 | None                                     |
       | L     | shetiphiancore           | 3.5.9                   | shetiphiancore-1.12.0-3.5.9.jar                     | None                                     |
       | L     | multibeds                | 1.4.7                   | multibeds-1.12.0-1.4.7.jar                          | None                                     |
       | L     | naturescompass           | 1.8.5                   | NaturesCompass-1.12.2-1.8.5.jar                     | None                                     |
       | L     | pamsimpleharvest         | 2.0.0                   | pamsimpleharvest-2.0.0.jar                          | None                                     |
       | L     | pathundergates           | 1.12.2-3.0.0-release    | pathundergates-1.12.2-3.0.0-release.jar             | None                                     |
       | L     | placebo                  | 1.6.0                   | Placebo-1.12.2-1.6.0.jar                            | None                                     |
       | L     | platforms                | 1.4.6                   | platforms-1.12.0-1.4.6.jar                          | None                                     |
       | L     | plp                      | 1.0                     | PLP_0.14.jar                                        | None                                     |
       | L     | pvj                      | 1.6.4                   | ProjectVibrantJourneys-1.12.2-1.6.4.jar             | None                                     |
       | L     | rsgauges                 | 1.2.5                   | rsgauges-1.12.2-1.2.5.jar                           | None                                     |
       | L     | signpost                 | 1.08.5                  | signpost-1.12.2-1.08.5.jar                          | None                                     |
       | L     | soundfilters             | 0.12.1_for_1.12         | SoundFilters-0.12.1_for_1.12.jar                    | None                                     |
       | L     | streams                  | 0.4.9                   | Streams-1.12-0.4.9.jar                              | None                                     |
       | L     | surge                    | 2.0.77                  | Surge-1.12.2-2.0.77.jar                             | None                                     |
       | L     | texfix                   | 4.0                     | TexFix V-1.12-4.0.jar                               | None                                     |
       | L     | toastcontrol             | 1.8.1                   | Toast Control-1.12.2-1.8.1.jar                      | None                                     |
       | L     | toolbelt                 | 1.9.14                  | ToolBelt-1.12.2-1.9.14.jar                          | None                                     |
       | L     | torchmaster              | 1.8.5.0                 | torchmaster_1.12.2-1.8.5.0.jar                      | None                                     |
       | L     | unuparts                 | 5.0.1                   | UNU Parts Pack [MTS] 1.12.2-20.4.0-5.0.1.jar        | None                                     |
       | L     | unucivil                 | 5.0.3                   | UNU Civilian Pack [MTS] 1.12.2-20.4.0-5.0.3.jar     | None                                     |
       | L     | vanillafix               | 1.0.10-150              | VanillaFix-1.0.10-150.jar                           | None                                     |
       | L     | walkman                  | 1.0                     | walkman-1.12.2-1.0.jar                              | None                                     |
       | L     | wawla                    | 2.6.275                 | Wawla-1.12.2-2.6.275.jar                            | None                                     |
       | L     | wearablebackpacks        | 3.1.4                   | WearableBackpacks-1.12.2-3.1.4.jar                  | None                                     |
       | L     | worldedit                | 6.1.10                  | worldedit-forge-mc1.12.2-6.1.10-dist.jar            | None                                     |
       | L     | worldeditcuife3          | 3.0.9                   | WorldEdit-CUI-FE3-1.12.2-3.0.9.jar                  | None                                     |
       | L     | wrcbe                    | 2.3.2                   | WR-CBE-1.12.2-2.3.2.33-universal.jar                | None                                     |
       | L     | betteranimals            | 5.5.0                   | betteranimals-1.12.2-5.5.0.jar                      | None                                     |
       | L     | immersiveintelligence    | 0.2.0.1                 | immersiveintelligence-0.2.0.1.jar                   | None                                     |
       | L     | industrialwires          | 1.7-39                  | IndustrialWires-1.7-39.jar                          | None                                     |
       | L     | jade                     | 0.1.0                   | Jade-0.1.0.jar                                      | None                                     |
  Loaded coremods (and transformers): IELoadingPlugin (ImmersiveEngineering-core-0.12-98.jar)
                                        blusunrize.immersiveengineering.common.asm.IEClassTransformer
                                      EngineersDoorsLoadingPlugin (engineers_doors-1.12.2-0.9.1.jar)
                                        nihiltres.engineersdoors.common.asm.EngineersDoorsClassTransformer
                                      ForgelinPlugin (Forgelin-1.8.4.jar)
                                        
                                      Do not report to Forge! (If you haven't disabled the FoamFix coremod, try disabling it in the config! Note that this bit of text will still appear.) (foamfix-0.10.14-1.12.2.jar)
                                        pl.asie.foamfix.coremod.FoamFixTransformer
                                      SecurityCraftLoadingPlugin ([1.12.2] SecurityCraft v1.9.0.2.jar)
                                        
                                      CTMCorePlugin (CTM-MC1.12.2-1.0.2.31.jar)
                                        team.chisel.ctm.client.asm.CTMTransformer
                                      EntityCullingPlugin (EntityCulling-1.12.2-4.2.5.jar)
                                        meldexun.entityculling.asm.EntityCullingClassTransformer
                                      FutureMC (future-mc-0.2.8.jar)
                                        thedarkcolour.futuremc.asm.CoreTransformer
                                      FarseekCoreMod (Farseek-1.12-2.5.1.jar)
                                        farseek.core.FarseekClassTransformer
                                      ShetiPhian-ASM (ShetiPhian-ASM-1.12.0.jar)
                                        shetiphian.asm.ClassTransformer
                                      VanillaFixLoadingPlugin (VanillaFix-1.0.10-150.jar)
                                        
                                      CreativePatchingLoader (CreativeCore_v1.10.69_mc1.12.2.jar)
                                        
                                      JustEnoughIDs Extension Plugin (JustEnoughIDs-1.0.4-SNAPSHOT-thin.jar)
                                        org.dimdev.jeid.JEIDTransformer
                                      IILoadingPlugin (immersiveintelligence-core-0.2.0.1.jar)
                                        pl.pabilo8.immersiveintelligence.common.asm.IIClassTransformer
                                      PUGFMLLoadingPlugin (pathundergates-1.12.2-3.0.0-release.jar)
                                        cyberslas.pathundergates.asm.GrassPathTransformer
                                      DoubleSlabs Plugin (DoubleSlabs-1.12-0.12.2.jar)
                                        cjminecraft.doubleslabs.client.asm.ClassTransformer
                                      SurgeLoadingPlugin (Surge-1.12.2-2.0.77.jar)
                                        
                                      Better Biome Blend (betterbiomeblend-1.12.2-1.1.7-forge.jar)
                                        
                                      LittlePatchingLoader (LittleTiles_v1.5.57_mc1.12.2.jar)
                                        com.creativemd.littletiles.LittleTilesTransformer
  GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.6.0 NVIDIA 471.68' Renderer: 'NVIDIA GeForce GTX 1070/PCIe/SSE2'
  Suspected Mods: Unknown
  Launched Version: forge-14.23.5.2860
  LWJGL: 2.9.4
  OpenGL: NVIDIA GeForce GTX 1070/PCIe/SSE2 GL version 4.6.0 NVIDIA 471.68, NVIDIA Corporation
  GL Caps: Using GL 1.3 multitexturing.
           Using GL 1.3 texture combiners.
           Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
           Shaders are available because OpenGL 2.1 is supported.
           VBOs are available because OpenGL 1.5 is supported.
  Using VBOs: Yes
  Is Modded: Definitely; Client brand changed to 'fml,forge'
  Type: Client (map_client.txt)
  Resource Packs: 
  Current Language: English (US)
  Profiler Position: N/A (disabled)
  CPU: 8x Intel(R) Core(TM) i3-10100F CPU @ 3.60GHz

**Comment by Mysticpasta1** [comment]
Try to install MixinBootstrap with it
should be in the Relations Tab on CF

**Comment by pyroAvatar91** [comment]
I installed MixinBoostrap and nothing changed. However, I figured out the problem is also related to Vanilla Fix, which seems to be connected with what I read of another post: #121 
The error no longer occurs when I disable VanillaFix. I have not yet tried downgrading JEID as suggested in the referenced post; I have some mods that really rely on JEID, so I would rather just do without VanillaFix until this issue is resolved.
